web-client/src/views/DocketRecord/DocketRecord.jsx

```diff
@@ -104,7 +104,7 @@ export const DocketRecord = connect(
                     <td>
                       <span
                         className={classNames(
-                          entry.isStricken && 'stricken-docket-record',
+                          entry.isStricken && 'stricken-ustc-table',
```

Comment:
This doesn't look right - but maybe it is. Just thought I'd note it!

Reply:
Good catch! I think that's not right.